### DISCUSSION TITLE
Total beginner, confused and lost, please help!

### DISCUSSION THREAD

**Discussion by BasedAnon** [NVIDIAGameWorks/kaolin#732] [General]
What is Kaolin?  It looks so cool, but what is its use case?
(my guess is that it's for building generative models for 3D objects)
Can I find example datasets to play around with?  I don't have any of my own, and I can't find any...

Also, no one is talking in the forums, or here...  Is this vaporware?  Because it shows every sign of it...  o_O

**Comment by itsvismay** [comment]
Hi @BasedAnon !

Kaolin is a library of tools for working with a variety of 3D representations. 
There are many use cases for Kaolin, ranging from mesh operations, to differentiable rendering to fast, interactive, representation agnostic physics.

Here are some jupyter notebook tutorials for using Kaolin: https://kaolin.readthedocs.io/en/latest/notes/tutorial_index.html
